README.md

```diff
@@ -1 +1,174 @@
-# Budget_APP
\ No newline at end of file
+## Budget App
+
+<a name="readme-top"></a>
+
+# ￼ Table of Contents
+
+- [￼ About the Project](#about-project)
+- [🛠 Built With](#built-with)
+- [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+- ￼? Getting Started](#getting-started)
+- [Setup](#setup)
+- [Prerequisites](#prerequisites)
+- [Install](#install)
+- [Usage](#usage)
+- [Run tests](#run-tests)
+- [Deployment](#triangular_flag_on_post-deployment)
+- ￼? Authors](#authors)
+- ￼? Future Features](#future-features)
+- ￼? Contributing](#contributing)
+- ￼⭐️ Show your support](#support)
+- ￼? Acknowledgements](#acknowledgements)
+- ￼? License](#license)
+
+
+#￼? [Budget App] <a name="about-project"></a>
+
+
+**[Budget App]** is a mobile web application where you can manage your budget: You have a list of expenses associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+```sh
+  Ruby on Rails
+```
+
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **A user can register and login**
+- **Added authourization**
+- **A user see the money spent on each category.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Live Demo
+
+[Budget App](https://budget-tracker-7y1z.onrender.com/)
+
+## video presentation
+
```

Comment:
- [ ] Dear @lawrahkonwea, Thanks for sharing the access to the video over google drive, But we need to make sure the video is accessible to the public, Please make sure anyone can access and watch the documentation video.